services/backend-notification-pusher/src/Wire/BackendNotificationPusher/Options.hs

```diff
@@ -0,0 +1,24 @@
+module Wire.BackendNotificationPusher.Options where
+
+import Data.Aeson
+import Data.Domain
+import Imports
+import Util.Options
+
+data Opts = Opts
+  { federatorInternal :: !Endpoint,
+    rabbitMQ :: !RabbitMQOpts,
+    remoteBackends :: [Domain]
```

Comment:
this would need to be handled the same way we do things in #3235, but until we've decided that this is fine i suppose.

Reply:
Yes, that's the plan. If you finish #3235 before this PR, we get to implement the change here (as I guess some test would break otherwise). If we finish this PR before you, then you get to implement it in yours. So its a race :rofl: